src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2023 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { columns, data, rowKey } from '../Table/Table.mocks'
+import { Button } from '../Button'
+import { Card } from '.'
+import { Table } from '../Table'
+import { actionButton } from './Card.mocks'
+
+const meta = {
+  component: Card,
+  args: {
+    ...Card.defaultProps,
+    children: 'Card Content',
+    docLink: 'https://mia-platform.eu',
+    extra: <Button>{'Button'}</Button>,
+    subtitle: 'Card Subtitle',
+    title: 'Card Title',
```

Comment:
It would be nice to reuse the same props also in tests. I would create a mocks file with common definitions. 
```
export const extra = <Button>{'Button'}</Button>
```

Reply:
I would also assign these to the Default story, so we can define other stories in the opposite way (Title story will have title prop instead of undefined props)